proto/viam/service/slam/v1/slam.proto

```diff
@@ -123,6 +145,30 @@ message GetInternalStateRequest {
 }
 
 message GetInternalStateResponse {
-  // Internal state of the SLAM algorithm required to continue mapping/localization
+  // A chunk of the internal state of the SLAM algorithm required to continue
+  // mapping/localization
   bytes internal_state = 1;
 }
+
+message GetPointCloudMapStreamRequest {
+  // Name of slam service
+  string name = 1;
+}
+
+message GetPointCloudMapStreamResponse {
+  // Pointclouds are returned in chunks as a set of bytes which can be
+  // aggregated into the standard PCD format
```

Comment:
should be a bit more specific in the wording. I think it may just be concatenating the responses from the beginning and end of the stream and nothing in between.